pyproject.toml

```diff
@@ -17,3 +36,27 @@ exclude_lines = [
 
 [tool.pytest.ini_options]
 pythonpath = ["postcode_lookup"]
+
+[tool.uv.sources]
+dc-response-builder = { git = "https://github.com/DemocracyClub/dc_response_builder.git", rev = "1.0.0" }
+
+[dependency-groups]
+dev = [
+    "aws-sam-cli==1.119.0",
+    "beautifulsoup4==4.12.3",
+    "ipdb==0.13.13",
+    "pydantic-factories==1.17.3",
+    "pydantic[email]==1.10.17",
+    "pytest==8.2.2",
+    "respx==0.21.1",
+    "uvicorn==0.30.1",
+    "ruff==0.4.10",
+    "pytest-cov==5.0.0",
+    "coverage[toml]==7.5.4",
+    "pytest-asyncio==0.23.7",
+    "pytest-playwright==0.5.0",
+    "pre-commit==3.7.1",
+    "freezegun==1.5.1",
+    "playwright==1.45.1",
+    "beautifulsoup4==4.12.3",
+]
```

Comment:
This is an application, so can we also set

```
[tool.uv]
package = false
```

Somewhere in here.

Reply:
https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/596fe884c6676d76d69486cba70eddc89ce65188